pkg/kubernetes/kubernetes.go

```diff
@@ -162,7 +163,8 @@ func (k Kubernetes) ExecWithOptions(options ExecOptions) (bytes.Buffer, error) {
 
 // FindKubeConfig returns local Kubernetes configuration
 func FindKubeConfig() string {
```

Comment:
This function is only ever called from test code, so we can move this to `test/e2e/utils/kubernetes.go`. It's also the only consumer of `DefaultKubeConfig`, so I would just make the method:

```go
// FindKubeConfig returns local Kubernetes configuration
func FindKubeConfig() string {
	config, exists := os.LookupEnv("KUBECONFIG")
	if exists {
		return config
	}
	homedir, err := os.UserHomeDir()
	ExpectNoError(err)
	return homedir + "/.kube/config"
}
```

I avoided using `consts.GetEnvWithDefault` as the above code will start pass if the tests are run in an environment where the home dir is not configured.

Reply:
great idea thanks

Reply:
It seems that `ExpectNoError(err)` is a test call. Isn't it?

Reply:
Yes, but if we move this function to `test/e2e/utils/kubernetes.go` that's not a problem

Reply:
According to the call hierarchy it seems that the function is called only from the test code... maybe we can move it to there.

Reply:
I said something wrong... it is used also my the main code. Maybe we can safely ignore an error of the call of `os.UserHomeDir()`, since with this implementation we only run it in case the `KUBECONFIG` system property is not set.

Reply:
> I said something wrong... it is used also my the main code.

Where? The only reference I see in production code is from the function's own comment

Reply:
You're right... moving it :P